subgraphs/across-v2/src/handlers/liquidity.ts

```diff
@@ -61,8 +52,21 @@ export function handleLiquidityAdded(event: LiquidityAdded): void {
   } else {
     outputTokenAddress = hubPoolContractCall.value.getLpToken();
   }
-  // const outputToken = sdk.Tokens.getOrCreateToken(outputTokenAddress);
-  // pool.addOutputTokenSupply(event.params.lpTokensMinted);
+  const outputToken = sdk.Tokens.getOrCreateToken(outputTokenAddress);
+
+  // pool
+  const poolId = event.address
+    .toHexString()
+    .concat(event.params.l1Token.toHexString());
+  const pool = sdk.Pools.loadPool<string>(Bytes.fromUTF8(poolId));
+
+  if (!pool.isInitialized) {
+    pool.initialize(token.name, token.symbol, BridgePoolType.LIQUIDITY, token);
+    pool.pool.outputToken = outputToken.id;
+    pool.pool.save();
```

Comment:
It doesn't make a difference in this case, but preferable do `pool.save()`.

The wrapper could potentially have some additional pre/post save logic that you'd be skipping :P

Reply:
As a note assemblyscript doesn't respect access modifiers, so it will still compile and run.

Reply:
Ah, didn't know. Why do we use access modifiers in SDK then? For readability?